### PR TITLE
[Fall-fest]: prepare fall fest page 2023

### LINKED ISSUE #3518: Create the new fall fest page

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Closes #3518 

## Implementation details

- use `/fall-fest` page for 2023 event
- add Fall Fest 2023 content file
- remove University Directory until further instruction
- add event image, and intro copy
- remove agenda content, and Schedule of Events section until further instruction

## Preview
https://qiskit-org-pr-3525.dcq4xc5i083.us-south.codeengine.appdomain.cloud/events/fall-fest

### CODE REVIEW

File: pages/events/fall-fest.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      </section>
+    </div>
+  </article>
+</template>
+
+<script setup lang="ts">
+import { header, helpfulResources } from "~/constants/fallFest2023Content";
+
+definePageMeta({
+  pageTitle: "Qiskit Fall Fest 2023",
```
Comment (y4izus) [inline pages/events/fall-fest.vue:54]:
```suggestion
  pageTitle: headerData.titleLine1,
```

File: pages/events/fall-fest.vue [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  routeName: "fall-fest",
+});
+
+const config = useRuntimeConfig();
+
+const title = "Qiskit Fall Fest 2023";
```
Comment (y4izus) [inline pages/events/fall-fest.vue:60]:
```suggestion
const title = headerData.titleLine1;
```

File: pages/events/fall-fest.vue [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    <LayoutLeadSpaceWithCard
+      :cta="headerData.cta"
+      :card-title="headerData.cardSectionHeading"
+    >
+      <template #title>
+        {{ headerData.titleLine1 }}
+      </template>
+      <template #description>
+        <p>
+          The Qiskit Fall Fest is a collection of quantum computing events on
```
Comment (y4izus) [inline pages/events/fall-fest.vue:12]: You can add this description to the content file under `header`, with the rest of the content

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank you! I just have small request

File: pages/events/fall-fest.vue (hunk 3 above)
Comment (techtolentino) [reply pages/events/fall-fest.vue]: Addressed [here](https://github.com/Qiskit/qiskit.org/pull/3525/commits/f097177d9aa8d68f1b2a9f1a422b786226ef4f0f)

**Review by y4izus — APPROVED** [review]
LGTM!

**Pull request merged**